pages/blog/hacktoberfest-2022.md

```diff
@@ -0,0 +1,208 @@
+        <img src="https://avatars.githubusercontent.com/nawed2611" width="100px;" alt=""/>
+        <br />
+        <sub>
+          <b>
+            nawed2611 -  1 PR
+          </b>
+        </sub>
+      </a>
+    </td>
+     <td align="center">
+      <a href="https://github.com/dhfherna">
+        <img src="https://avatars.githubusercontent.com/dhfherna" width="100px;" alt=""/>
+        <br />
+        <sub>
+          <b>
+           dhfherna - 1 PR
+          </b>
+        </sub>
+      </a>
+    </td>
+         <td align="center">
+      <a href="https://github.com/amit-ksh">
+        <img src="https://avatars.githubusercontent.com/amit-ksh" width="100px;" alt=""/>
+        <br />
+        <sub>
+          <b>
+           amit-ksh - 1 PR
+          </b>
+        </sub>
+      </a>
+    </td>
+    </tr>
+          <tr>
+    <td align="center">
+      <a href="https://github.com/mastdev">
+        <img src="https://avatars.githubusercontent.com/mastdev" width="100px;" alt=""/>
+        <br />
+        <sub>
+          <b>
+          mastdev -  2 PRs
+          </b>
+        </sub>
+      </a>
+    </td>
+        <td align="center">
+      <a href="https://github.com/codingtenshi">
+        <img src="https://avatars.githubusercontent.com/codingtenshi" width="100px;" alt=""/>
+        <br />
+        <sub>
+          <b>
+            codingtenshi -  4 PRs
+          </b>
+        </sub>
+      </a>
+    </td>
+     <td align="center">
+      <a href="https://github.com/akshatnema">
+        <img src="https://avatars.githubusercontent.com/akshatnema" width="100px;" alt=""/>
+        <br />
+        <sub>
+          <b>
+           akshatnema - 1 PR
+          </b>
+        </sub>
+      </a>
+    </td>
+         <td align="center">
+      <a href="https://github.com/Mayaleeeee">
+        <img src="https://avatars.githubusercontent.com/Mayaleeeee" width="100px;" alt=""/>
+        <br />
+        <sub>
+          <b>
+           Mayaleeeee - 4 PRs
+          </b>
+        </sub>
+      </a>
+    </td>
+    </tr>
+</table>
+
+## Late to the party?
+
+Not a problem. We've got tons of issues you can explore and start contributing to in no time. Because we are a community-first open-source project, we've made it our responsibility to make it easier for contributors to navigate through issues easily. That's why we've built the AsyncAPI [community dashboard](https://www.asyncapi.com/community/dashboard) to help you explore some exciting good first issues.
+
+## Things to learn from
+
+We were overwhelmed and weren't fully prepared to handle the enthusiastic response from the community. Because we needed to curate more issues tagged Hacktoberfest, many of the issues were claimed before the mid of October, which left other contributors interested left out because of the lack of more issues.
+
+Once beaten twice shy. In other to avoid these potential pitfalls next year, we're gonna put some things in place, which include the following.
```

Comment:
```suggestion
Once beaten twice shy. In order to avoid these potential pitfalls next year, we're gonna put some things in place, which include the following.
```